webpack.embedding-sdk.config.js

```diff
@@ -145,8 +145,26 @@ module.exports = env => {
       }),
       new webpack.EnvironmentPlugin({
         EMBEDDING_SDK_VERSION,
+        GIT_BRANCH: require("child_process")
+          .execSync("git rev-parse --abbrev-ref HEAD")
+          .toString()
+          .trim(),
+        GIT_COMMIT: require("child_process")
+          .execSync("git rev-parse HEAD")
+          .toString()
+          .trim(),
         IS_EMBEDDING_SDK: true,
       }),
+      new webpack.DefinePlugin({
+        BUILD_TIME: webpack.DefinePlugin.runtimeValue(
+          () => JSON.stringify(new Date().toISOString()),
+          true, // This flag makes it update on each build
+        ),
+        "process.env.BUILD_TIME": webpack.DefinePlugin.runtimeValue(
+          () => JSON.stringify(new Date().toISOString()),
+          true,
+        ),
```

Comment:
just curious why we need to define both `BUILD_TIME` and `process.env.BUILD_TIME`?

Reply:
ahah we don't, i was experimenting, lemme fix that

Reply:
fixed 078e5bc0a4ccb9f61c9fac4ad84eb764a4dbd5cc